pkg/api/helpers.go

```diff
@@ -229,7 +228,7 @@ func InspectIdentifiers(api *ScalewayAPI, ci chan ScalewayResolvedIdentifier, cj
 			if len(idents.Identifiers) == 0 {
 				log.Errorf("Unable to resolve identifier %s", idents.Needle)
 			} else {
-				showResolverResults(idents.Needle, idents.Identifiers)
+				logrus.Fatal(showResolverResults(idents.Needle, idents.Identifiers))
```

Comment:
oh nice, you actually moved the logrus.fatal inside the goroutine :) ignore my last comment about this